src/material/schematics/ng-generate/table/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.ts

```diff
@@ -16,9 +16,9 @@ import { <%= classify(name) %>DataSource, <%= classify(name) %>Item } from './<%
   changeDetection: ChangeDetectionStrategy.<%= changeDetection %><% } %>
 })
 export class <%= classify(name) %>Component implements AfterViewInit, OnInit {
-  @ViewChild(MatPaginator) paginator: MatPaginator;
-  @ViewChild(MatSort) sort: MatSort;
-  @ViewChild(MatTable) table: MatTable<<%= classify(name) %>Item>;
+  @ViewChild(MatPaginator, {static: true}) paginator: MatPaginator;
+  @ViewChild(MatSort, {static: true}) sort: MatSort;
+  @ViewChild(MatTable, {static: true}) table: MatTable<<%= classify(name) %>Item>;
```

Comment:
These shouldn't need to be static since they're not used until `afterViewInit`

Reply:
Yes also we generally should avoid making these static as it make the the component less flexible otherwise. e.g. if someone adds `*ngIf` around the generated code, the queries would no longer work.

Static `true` should only be used if it is strictly necessary (e.g. for embedded view creation)

Reply:
Sure, I misunderstood the error message. It said _0 arguments passed, expected 1-2_ or something like that, because `MatTable` was not `import`ed.

I'll remove the `static` options.